app/main.py

```diff
@@ -1,2 +1,65 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.items = []
```

Comment:
It seems that it's a redundant list. It just requires extra memory and time to add/remove elements